### DISCUSSION TITLE
I cannot download for the following reasons

### DISCUSSION THREAD

**Discussion by wjeie** [aclap-dev/video-downloadhelper#510] [Help and Q&A]
### Your question

I can't download due to at as.receive (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:13157) at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42624

What do I need to do to download?

### Addon & CoApp version and details

8.2.2.9 (stable)

**Comment by GrampaWildWilly** [comment]
You need to fill out the full information about your versions of the software by following the instructions on the page where you created this problem report.  You should edit your post to provide this information.  Just click the 3 dots above & to the right of your post.  You should also supply the URL of the web page on which your problematic video is hosted.
